fs/log/log.go

```diff
@@ -114,8 +118,27 @@ func InitLogging() {
 
 	fs.LogPrintPid = strings.Contains(flagsStr, ",pid,")
 
+	if Opt.FilenameTimeFormat != "" {
+		tmp := strings.SplitN(Opt.FilenameTimeFormat, "~#~", 2)
+		if len(tmp) != 2 {
+			log.Fatalf("Invlaid Filename Time Formart provided via --file-time-format. Please refer docs for proper usage.")
```

Comment:
typos -- `Invalid` and `Format`

Reply:
Ahhh, for some reason JB IDE didn't point out typos in line. Can't believe I made such a stupid mistake. My bad. Fixed the typos.

Thanks for pointing out.